tensorflow/python/ops/image_ops_test.py

```diff
@@ -164,6 +164,80 @@ def testShapeInference(self):
       self.assertFalse(rgb_unknown.get_shape())
 
 
+class AdjustGamma(test_util.TensorFlowTestCase):
```

Comment:
- Add corresponding tests in `image_ops_test.py` according to [scikit-image](https://github.com/scikit-image/scikit-image/blob/master/skimage/exposure/tests/test_exposure.py#L308)
  - test_adjust_gamma_one
  - test_adjust_gamma_zero
  - test_adjust_gamma_less_one
  - test_adjust_gamma_greater_one